SU-SWS/stanford_news
# D8CORE-3994: fixing the specificity of the article h2 selector

# READY FOR REVIEW

# Summary
- When on a page, you had a news list card and a regular card, there was a selector that overrode the su-card h2 margins. 

# Needed By (Date)
- 4/30

# Urgency
- high

# Steps to Test

1.  First create a page with a List paragraph that has a News Card. 
3. Then add a card object that has a headline.
4. notice that the cards now have a funny extra margin left on the headline.
5. Remove the list item and see the card headline is fixed.
6. Add it back.
7. Pull in this change and clear caches.
8. Notice that you can have both without the su-card being effected. 

# Affected Projects or Products
- News
- SBT
- SOE - Tested
- SAA - Tested

# Associated Issues and/or People
- [D8CORE-3994](https://stanfordits.atlassian.net/browse/D8CORE-3994)
- Fyi @rebeccahongsf 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
